final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +113,90 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  'name' => 'Really Big Foot',
+  'smell' => 'mold',
+  'weight' => 540,
+  'citiesDestroyed' => ['Jericho', 'Persepolis', 'Aphek'],
+  'luckyNumbers' => [13, 96, 800],
+  'addrees' => {
+    'number' => 23,
+    'street' => 'Gremlin Ave',
+    'state' => 'New Jersey',
+    'zip' => 12666
+  }
+}
+puts scary_monster
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    
+    @@arch_nemesis = "The Syntax Error"
```

Comment:
Curious why you are initializing with class variables (`@@`) rather than instance variables (`@`) here?